sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare_async.py

```diff
@@ -438,4 +439,18 @@ async def test_normalized_text(self, client):
             e for e in result[0].entities if hasattr(e, "normalized_text")
         ])
 
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_healthcare_assertion(self, client):
+        response = await (await client.begin_analyze_healthcare_entities(
+            documents=["Baby not likely to have Meningitis."]
+        )).result()
+
+        result = []
+        async for r in response:
+            result.append(r)
 
+        # currently can only test certainty
+        # have an issue to update https://github.com/Azure/azure-sdk-for-python/issues/17088
+        # meningitis_entity = next(e for e in result[0].entities if e.text == "Meningitis")
+        # assert meningitis_entity.assertion.certainty == EntityCertainty.NEGATIVE
```

Comment:
commented out code checking the certainty. This is due to a mismatch between what the swagger has defined as the casing, and the actual casing sent over the wire.

Have a [PR ](https://github.com/Azure/azure-rest-api-specs/pull/13277)out for this, blocked on checking what the desired behavior is, @abhahn is looking into this (thanks @abhahn !)